checker/examples/units-extension/Demo.java

```diff
@@ -1,5 +1,6 @@
 import org.checkerframework.checker.units.UnitsTools;
 import org.checkerframework.checker.units.qual.*;
+import qual.*;
```

Comment:
Nothing else changed in this file. Why do you need the additional import?
Can you replace both import * with specific imports?